packages/editor/src/components/post-saved-state/test/__snapshots__/index.js.snap

```diff
@@ -4,7 +4,9 @@ exports[`PostSavedState returns a disabled button if the post is not saveable 1`
 <button
   aria-disabled="true"
   aria-label="Save draft"
-  class="components-button is-compact is-tertiary has-icon"
+  class="components-button components-tooltip undefined is-compact is-tertiary has-icon"
```

Comment:
This `undefined` classname seems off.  Is it signaling a poor use of the `className` prop inside `<PostSavedState />`?

Reply:
Well spotted — that was a consequence of using the `useContext` hook, which actually made me realize that we should probably forward the `restProps` to the tooltip, and not the anchor. Worked on it in [`d52af82` (#57202)](https://github.com/WordPress/gutenberg/pull/57202/commits/d52af821c9e4b7db237c997b7b8a70eba10709c1), which also allows us to revert all of the snapshot changes